feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt

```diff
@@ -0,0 +1,169 @@
+                )
+            ) {
+                /*TODO here will be a nav component*/
+            }
+            Box(
+                modifier = Modifier
+                    .fillMaxWidth(0.75f)
+                    .padding(horizontal = sizes.minorSmallSize, vertical = sizes.minorRegularSize)
+            ) {
+                Text(
+                    text = stringResource(id = R.string.profile),
+                    fontSize = 22.sp,
+                    color = MaterialTheme.colorScheme.onPrimaryContainer
+                )
+            }
+            Column(
+                modifier = Modifier.padding(all = sizes.nothingSize),
+                horizontalAlignment = Alignment.End
+            ) {
+                LpEditFloatingActionButton(
+                    color = MaterialTheme.colorScheme.inversePrimary,
+                    colors = MaterialTheme.colorScheme.inversePrimary,
+                    elevation = FloatingActionButtonDefaults.elevation(sizes.nothingSize),
+                ) {}
+            }
+        }
+        Row(modifier = Modifier.fillMaxWidth()) {
+            Column(modifier = Modifier.padding(horizontal = sizes.smallSize)) {
+                /*Image(
+                    painter = userPhoto,
+                    contentDescription = stringResource(R.string.user_photo),
+                    contentScale = ContentScale.Crop,
+                    modifier = Modifier
+                        .size(sizes.extraSize78)
+                        .clip(CircleShape)
+                )*/
+                /*TODO Here will be the image implementation*/
+            }
+            Column() {
+                Box(modifier = Modifier.padding(vertical = sizes.midSmallSize)) {
+                    Text(
+                        text = "Juan Islas",
+                        fontWeight = FontWeight.Medium,
+                        fontSize = 18.sp,
+                        color = MaterialTheme.colorScheme.onPrimaryContainer
+                    )
+                }
+                Box(modifier = Modifier.padding(top = sizes.stroke)) {
+                    Text(text = "Apartment #101", color = MaterialTheme.colorScheme.tertiary)
+                }
+            }
+        }
+
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(
+                    start = sizes.regularSize,
+                    end = sizes.regularSize,
+                    top = sizes.smallSize,
+                    bottom = sizes.smallerSize
+                )
+        ) {
+            Icon(
+                Icons.Outlined.Phone,
+                contentDescription = stringResource(id = R.string.phone_icon),
+                modifier = Modifier.padding(end = sizes.minorSmallSize),
+                tint = MaterialTheme.colorScheme.tertiary
+            )
+            Text(
+                text = "(404) 979-2400",
+                fontWeight = FontWeight.Normal,
```

Comment:
please check the styles of all the texts because isn't look same as the design

Reply:
Text components will be handle as a UI component, as the buttons or cards, an specific file will be created, the task in jira is already created but is not assigned yet. Please check with Irving :)